src/common/types/store.ts

```
@@ -82,9 +82,13 @@ export const modifyProductSchema = productSchema.pick({
   additionalEmailText: true,
 }).partial().extend({
   verifiedIdentityRequired: z.boolean().optional(), // Override to remove the default
-});
+}).refine(
```

ESLint

[error] Replace `})` with `··})⏎··`

(prettier/prettier)